CSS/index.css

```diff
@@ -13,39 +13,36 @@ header {
     top: 0;
     width: 100%;
     background-color: rgb(9, 116, 119);
-    z-index: 999; 
+    z-index: 999;
     height: 160px;
     padding: 10px 0;
 }
 
 header nav {
     background-color: rgb(9, 116, 119);
-    
 }
```

Comment:
**Optional:** This is more of a stylistic suggestion. Add blank lines between all blocks in your CSS files for consistency. It might also help improve readability to have a blank line between your blocks.